src/frontend/pages/blogs/BlogPage.js

```diff
@@ -0,0 +1,110 @@
+import React, { useCallback} from 'react';
+import styled from 'styled-components';
+import UnstyledButton from '../../components/Button';
+import PreviewTable from '../../components/PreviewTable';
+import TableCell from '@material-ui/core/TableCell';
+import useBlogInfo from '../../hooks/blogs'
+import { useHistory } from 'react-router-dom';
+
+const Button = styled(UnstyledButton)``;
```

Comment:
No need to change, but in the future, I think you can just do:

`import { UnstyledButton as Button } from '../../components/Button';`